packages/compass-aggregations/src/components/aggregations/aggregations.jsx

```diff
@@ -83,6 +83,8 @@ const mapStateToProps = (state) => ({
   setIsNewPipelineConfirm: state.setIsNewPipelineConfirm,
   updateViewError: state.updateViewError,
   workspace: state.workspace,
+  isReadonly: state.isReadonly,
+  preferencesReadOnly: state.preferencesReadOnly,
```

Comment:
I tried to find the place where these two are being consumed but couldn’t – what am I missing? `Aggregations` forwards its props as-is to `Pipeline`, but `Pipeline` doesn’t seem to do anything with them?

Reply:
You are right, removed.